proto/proto.lock

```diff
@@ -1831,33 +1831,34 @@
             ]
           },
           {
-            "name": "Table",
+            "name": "TableGroups",
```

Comment:
I am wondering where are the corresponding changes in blueprint.proto?

Reply:
This is a bit strange. It looks to me like the ordering has changed. If you compare what's been removed to what has been added, nothing has actually changed. Perhaps this is because I'm using a different version of protoc on my machine?